src/BabelConfig.js

```diff
@@ -37,15 +78,7 @@ class BabelConfig {
     static default() {
         return {
             cacheDirectory: true,
-            presets: [
-                [
-                    '@babel/preset-env',
-                    {
-                        modules: false,
-                        forceAllTransforms: true
-                    }
-                ]
-            ],
+            presets: ['@babel/preset-env'],
             plugins: [
                 '@babel/plugin-syntax-dynamic-import',
                 '@babel/plugin-proposal-object-rest-spread',
```

Comment:
It looks like this line is added optional. It may not be necessary for people using modern browsers.

And 85 to 90 lines